src/Simulation/Simulators.Tests/TestProjects/HoneywellExe/HoneywellExe.csproj

```diff
@@ -22,4 +22,11 @@
     </ItemGroup>
```

Comment:
Should these Q# projects have their name changed so they don't have "Exe" in them, considering their output type is "Library"?

Reply:
That's a good question... they were originally Exe projects that got changed to Library so they could all be linked together, but now that they aren't linked together they can be changed back to Exe.

Reply:
Actually, if you want them to output exe, I think you have to specify `<OutputType>Exe</OutputType>`. I think without this, it defaults to the 'library' behavior.

Reply:
Oh, whoops... I thought it was the other way around. Fixing now...